### ISSUE TITLE
Allow `settings.inherit.json` to filter XForms

### ISSUE THREAD

**Issue by mukesh2006** [medic/medic-conf#76]
When a standard project is setup, the TLs manually [delete the unused forms from the instance](https://docs.google.com/document/d/1ZLh_jM2ndlz7vBeM7MYQ93UvW7SW7Ey7av2pkdP4O2U/edit#heading=h.wrp5bqfxacgw). It would be great if we've filter option in `settings.inherit.json` for XForms similar to JSON forms. 

Related Issue: https://github.com/medic/medic-projects/issues/4452 and https://github.com/medic/medic-conf/issues/49 

cc @abbyad @shreyakb @jillrshah @alxndrsn

**Comment by alxndrsn** [comment]
@mukesh2006 what command are you running, and from what path, to upload the inherited forms?

**Comment by mukesh2006** [comment]
@alxndrsn Here's what i do to upload the latest settings of standard project (let's say `xyz-project`):

1. Change the `"inherit": "./node_modules/medic/config/standard"` to `"inherit": "../../medic-webapp/config/standard"` (i.e. my latest local repo of medic-webapp) in the file `settings.inherit.json` of the `xyz-project`. 

2. Run `medic-conf --instance xyz-project.app` command from the project folder of `medic-projects/xyz-project`.

3. Revert back the changes made in `settings.inherit.json`

This causes uploading all the `JSON forms` available in the standard. An option to select the `JSON forms` would help upload selected forms only.

**Comment by alxndrsn** [comment]
@mukesh2006 couldn't you just update the `package.json` for `xyz-project` to depend on the latest `medic-webapp`, and then the standard forms should be picked correctly from `"./node_modules/medic/config/standard"` as required?

**Comment by abbyad** [comment]
That would help, but I don't think resolves this issue. We would still need a way to specify which forms to exclude, or perhaps better, only the ones that you want to include. Is that what you are requesting @mukesh2006?

**Comment by mukesh2006** [comment]
@abbyad Yes, it'd be better if we have a way to specify which forms to include/upload (including is better option than excluding). Currently all the forms are uploaded and then have to manually delete the unwanted forms from the instance.

cc @alxndrsn
